src/coreclr/vm/codeman.cpp

```diff
@@ -3893,7 +4130,7 @@ BOOL EEJitManager::JitCodeToMethodInfo(
     if (pRangeSection->_flags & RangeSection::RANGE_SECTION_RANGELIST)
         return FALSE;
 
-    TADDR start = dac_cast<PTR_EEJitManager>(pRangeSection->_pjit)->FindMethodCode(pRangeSection, currentPC);
+    TADDR start = dac_cast<PTR_EECodeGenManager>(pRangeSection->_pjit)->FindMethodCode(pRangeSection, currentPC);
     if (start == (TADDR)0)
         return FALSE;
 
```

Comment:
What happens if we end up with `InterpreterCodeHeader` in this and other similar places?

Reply:
Thank you for spotting this. I have overlooked the usage of the `CodeHeader` explicitly in this method. It needs to be abstracted.

Reply:
Hmm, I can see I have missed couple of other places too. Let me rescan the usage of the `CodeHeader` to make sure it doesn't appear where it should not.

Reply:
Fixed